### ISSUE TITLE
NPE for bootstrap when javascript filtering is used

### ISSUE THREAD

**Issue by kevinclcn** [zendesk/maxwell#1288]
I used suppress statement in the javascript filtering, and when I bootstrap a huge amount of records, the bootstrap thread report NPE exception.

Here are the stack:

```
java.lang.NullPointerException
	at com.zendesk.maxwell.schema.PositionStoreThread.setPosition(PositionStoreThread.java:112)
	at com.zendesk.maxwell.MaxwellContext.setPosition(MaxwellContext.java:265)
	at com.zendesk.maxwell.producer.AbstractAsyncProducer.push(AbstractAsyncProducer.java:67)
	at com.zendesk.maxwell.producer.MaxwellKafkaProducerWorker.run(MaxwellKafkaProducer.java:213)
	at java.lang.Thread.run(Thread.java:748)
```

I suppose that the RowMap's position field is null.

![image](https://user-images.githubusercontent.com/1789301/60865043-c6a3f880-a257-11e9-8c73-13c07b5999b8.png)

**Comment by osheroff** [comment]
will look.

**Comment by osheroff** [comment]
fixed, lo these many months later, in v1.22.6
